plugins/dependencies/src/main/kotlin/dependencies/CashApp.kt

```diff
@@ -4,6 +4,13 @@ import org.gradle.kotlin.dsl.IsNotADependency
 
 object CashApp {
 
+    /**
+     * SQLDelight - Generates typesafe Kotlin APIs from SQL
+     *
+     * - [Official Website - API and documentation](https://cashapp.github.io/sqldelight/)
+     * - [cashapp/sqldelight: GitHub](https://github.com/cashapp/sqldelight)
+     * - [CHANGELOG](https://github.com/cashapp/sqldelight/blob/master/CHANGELOG.md)
```

Comment:
The style here is inconsistent with the existing dependency notations KDoc that can be found in the `Square` object.

Having a consistent style makes a better impression on users and values the effort in making the whole project right.

Happy to improve the style, but we need to discuss it and agree on a style to stick with.
I'll complete a contributing guide covering this as we might welcome further contributions there from the community, and we might forget about it ourselves.

I'm not repeating this comment to all the other KDocs in this PR, this one stands for them all.